src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static void getRandomColor() {

```

Comment:
The method getRandomColor() should return a value, but it is currently void. You should return the color fetched from the Color enum.